packages/web/lib/hooks/brightId.ts

```diff
@@ -0,0 +1,82 @@
+import { CONFIG } from 'config';
+import { PlayerFragmentFragment } from 'graphql/autogen/types';
+import { useEffect } from 'react';
+
+const BRIGHTID_CONTEXT = 'MetaGame';
+const DEEPLINK_ENDPOINT = `brightid://link-verification/${CONFIG.brightIdNodeUrl}/${BRIGHTID_CONTEXT}`;
+const UNIVERSAL_LINK_ENDPOINT = `${CONFIG.brightIdAppUrl}/link-verification/${CONFIG.brightIdNodeUrl}/${CONFIG.brightIdNodeUrl}/${BRIGHTID_CONTEXT}`;
+const VERIFICATION_ENDPOINT = `${CONFIG.brightIdAppUrl}/node/v5/verifications/${BRIGHTID_CONTEXT}`;
+const POLL_INTERVAL = 5000;
+
+export const useBrightIdStatus = ({
+  player,
+}: {
+  player: PlayerFragmentFragment;
+}): {
+  verified: boolean;
+  deeplink: string;
+  universalLink: string;
+} => {
+  const verified =
```

Comment:
Maybe you can wrap these 3 variables in an object that you compute with `useMemo` then return.